pkgs/by-name/de/decker/package.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, SDL2
+, SDL2_image
+, unixtools
+, multimarkdown
+}:
+
+stdenv.mkDerivation rec {
+  pname = "decker";
+  version = "1.31";
+
+  src = fetchFromGitHub {
+    owner = "JohnEarnest";
+    repo = "Decker";
+    rev = "v${version}";
+    hash = "sha256-9utCIf7LO/ms46QqagkcXZ3BuvRuLa6nE78MgkbaEjA=";
+  };
+
+  buildInputs = [
+    SDL2
+    SDL2_image
+    multimarkdown
+    unixtools.xxd
+  ];
+
+  doCheck = true;
+
+  postPatch = ''
+    patchShebangs ./scripts/*
+  '';
+
+  buildPhase = ''
+    make lilt
+    make decker
+    make docs
+  '';
```

Comment:
```suggestion
  buildPhase = ''
    runHook preBuild
    make lilt
    make decker
    make docs
    runHook postBuild
  '';
```

Reply:
I didn't notice that. Thank you for looking out.